packages/flutter_tools/test/integration.shard/overall_experience_test.dart

```diff
@@ -384,7 +384,8 @@ void main() {
             return null;
           }),
           Barrier('Performing hot restart...'.padRight(progressMessageWidth)),
-          Multiple(<Pattern>[RegExp(r'^Restarted application in [0-9]+ms.$'), 'called main', 'called paint'], handler: (String line) {
+          // This could look like 'Restarted application in 1,237ms.'
+          Multiple(<Pattern>[RegExp(r'^Restarted application in [0-9][0-9,]*ms.$'), 'called main', 'called paint'], handler: (String line) {
```

Comment:
Could also handle `10.1s` by not getting cute with the numbers.
```
^Restarted application in.*s.$
```

Reply:
will do...however I'd be happy to have this test fail if we could hot restart that fast

Reply:
oh never mind, this is seconds, I see what you mean

Reply:
done